src/appleseed-max-impl/appleseedsssmtl/appleseedsssmtl.cpp

```diff
@@ -828,3 +834,12 @@ HINSTANCE AppleseedSSSMtlClassDesc::HInstance()
 {
     return g_module;
 }
+
+bool AppleseedSSSMtlClassDesc::IsCompatibleWithRenderer(ClassDesc& rendererClassDesc)
```

Comment:
With our naming convention: `renderer_class_desc`.

This applies to other materials as well.